sycl/source/detail/config.def

```diff
@@ -24,6 +24,7 @@ CONFIG(SYCL_HOST_UNIFIED_MEMORY, 1, __SYCL_HOST_UNIFIED_MEMORY)
 CONFIG(SYCL_CACHE_DIR, 164, __SYCL_CACHE_DIR)
 CONFIG(SYCL_CACHE_TRACE, 1, __SYCL_CACHE_TRACE)
 CONFIG(SYCL_CACHE_DISABLE_PERSISTENT, 1, __SYCL_CACHE_DISABLE_PERSISTENT)
+CONFIG(SYCL_CACHE_PERSISTENT, 1, __SYCL_CACHE_PERSISTENT)
```

Comment:
if the default behavior is 'off', shouldn't this value be initialized to '0' ?

Reply:
this is not default value, it is size of var in-memory rep